src/components/AnimatedMapboxLayer.vue

```diff
@@ -22,9 +21,21 @@ function getCoordsFromBounds(bounds: LngLatBounds) {
   ]
 }
 
-interface MapboxLayerOptions {
+function isBoundsWithinBounds(innerBounds: LngLatBounds, outerBounds: LngLatBounds) {
```

Comment:
Maybe use [Turf's booleanWithin](http://turfjs.org/docs/#booleanWithin) to check if one bounds is within another bounds?

Reply:
Type definitions of `booleanWithin` are missing furthermore the `innerBounds` and `outerBounds` should be transformed into turf points and the function would have a similar amount of lines. This would have been the function, but it does not have an added value for me.
```
function isBoundsWithinBounds(innerBounds: LngLatBounds, outerBounds: LngLatBounds) {
  // Convert the bounds to Turf.js points
  const innerNW = point([innerBounds.getNorthWest().lng, innerBounds.getNorthWest().lat])
  const innerSE = point([innerBounds.getSouthEast().lng, innerBounds.getSouthEast().lat])
  const outerNW = point([outerBounds.getNorthWest().lng, outerBounds.getNorthWest().lat])
  const outerSE = point([outerBounds.getSouthEast().lng, outerBounds.getSouthEast().lat])

  const isInnerWithinOuter = booleanWithin(innerNW, outerNW) && booleanWithin(innerSE, outerSE)

  return isInnerWithinOuter
}
```